documentation/docs/04-loading.md

```diff
@@ -105,6 +105,8 @@ If the page you're loading has an endpoint, the data returned from it is accessi
 
 If you return a Promise from `load`, SvelteKit will delay rendering until the promise resolves. The return value has several properties, all optional:
```

Comment:
Should we no longer have this end with `:` since the next thing is the warning box?
```suggestion
If you return a Promise from `load`, SvelteKit will delay rendering until the promise resolves. The return value has several properties listed below, all of which are optional.
```